doc/design/embargo-mvp.md

```diff
@@ -0,0 +1,131 @@
+# Embargo Minimum Viable Product
+Due to time constraints, we are implementing the core functionality for embargoing dandisets ASAP.
+This document is dedicated to the minimum viable product.
+A subsequent document will describe the full suite of features.
+
+
+# User experience
+
+## Web
+When creating a new dandiset, users will have checkbox to choose whether or not the dandiset should be embargoed.
+If so, the user must also specify an NIH award number (https://era.nih.gov/files/Deciphering_NIH_Application.pdf).
+The award number is an optional argument that is passed to the create dandiset endpoint.
+The award number will be added as a new `Funder` on an automatically generated Contributor Organization in the metadata.
+
+Instead of the normal `Publish` button and version browser, the DLP will instead show a `Release` button.
+Clicking it will open a confirmation modal informing the user that they are unembargoing, all their data will be publicized, there is no undoing, and that it can take some time for large dandisets.
+Confirming will lock the dandiset for the duration of the release.
+Once the release finishes, the dandiset will be like any other unpublished draft dandiset.
+
+Once created, an embargoed dandiset is only visible or searchable to owners.
+Instead of the `draft` chip, an `embargoed` chip should be used instead.
+
+## CLI
+The CLI experience will be basically unchanged.
+Uploads to an embargoed dandiset will function exactly the same from an API perspective, the data simply goes to a different location.
+
+### TODO: how to browse/download embargoed zarr files?
+Credentials need to be generated, distributed, and used by the client.
+
+
+# Data storage
+Embargoed assets will be stored in a separate S3 bucket.
+This bucket is private and not browseable by the general public.
+
+The API server will use the embargo bucket in exactly the same way it uses the public bucket, but it will only store embargoed data.
+
+When releasing an embargoed dandiset, all asset data for that dandiset is copied to the public bucket.
+
+When uploading a new asset to an embargoed dandiset, the server will first check if that blob has already been uploaded publically.
+If so, the public blob will be used instead of uploading the data again to the embargo bucket.
```

Comment:
in case anyone who has questions about privacy based on hash checking, i would say that should not be a concern. the data in dandi are way too specific for bitwise things to match or for someone to try generating data that matches a hash.

Reply:
Sounds good. Resolving (unresolve if there's more to discuss).